app/controllers/posts_controller.rb

```diff
@@ -1,39 +1,28 @@
 class PostsController < ApplicationController
```

Comment:
Please add a route in the config/routes.rb file for creating like. User should be able to like posts.